src/view/prDocumentCommentProvider.ts

```
@@ -64,6 +64,28 @@ export class PRDocumentCommentProvider implements vscode.DocumentCommentProvider
 
 		return await this._prDocumentCommentProviders[params.prNumber].replyToCommentThread(document, range, commentThread, text, token);
 	}
+
+	async editComment(document: vscode.TextDocument, comment: vscode.Comment, text: string, token: vscode.CancellationToken): Promise<void> {
+		let uri = document.uri;
+		let params = fromPRUri(uri);
+
+		if (!this._prDocumentCommentProviders[params.prNumber]) {
+			throw new Error(`Couldn't find document provider`);

```

We are returning null in other places when lookup fails. But this is a void method. Should we just ignore the call, or throw an error?

---

We should throw, but can we make the error message actionable to the user?

---

we should throw an error

---

we should throw an error

---

throw an error

---

throw an error

---

throw `Error`

---

throw `Error`

---

throw `Error`

---

throw `Error`